crytic_compile/platform/hardhat.py

```diff
@@ -54,7 +54,9 @@ def hardhat_like_parsing(
     files = sorted(
         os.listdir(build_directory), key=lambda x: os.path.getmtime(Path(build_directory, x))
     )
-    files = [str(f) for f in files if str(f).endswith(".json")]
+    # files = [str(f) for f in files if str(f).endswith(".json")]
+    files = [f for f in files if not f.endswith(".output.json")]
```

Comment:
the original code picked only json files, the new function may pick non-json files as well. Is there a reason for this change? Also please clean the commented out code

Reply:
No you're right - this was a mistake from my end. Reverting back to having both filters.